package.json

```diff
@@ -19,7 +19,9 @@
   },
   "homepage": "https://github.com/majac91/Travel_journal#readme",
   "dependencies": {
-    "moment": "^2.27.0"
+    "moment": "^2.27.0",
+    "scripts": "^0.1.0",
+    "src": "^1.1.2"
```

Comment:
You can remove `scripts` and `src`. They are not needed in this project `npm uninstall scripts src --save`